EmployeeLib/Employee.cs

```diff
@@ -20,5 +21,10 @@ public virtual void CalculatePerHourRate(int rank)
 
             Salary = baseAmount + (rank * 2);
         }
+
+        public void GeneratePerformanceReview()
```

Comment:
this should not be responsibility from employee, only managers can generate performance review